packages/analytics/dashboard-renderer/src/components/BarChartRenderer.vue

```diff
@@ -1,21 +1,25 @@
 <template>
   <QueryDataProvider
     v-slot="{ data }"
-    :query="props.query"
-    :query-ready="props.queryReady"
+    :query="query"
+    :query-ready="queryReady"
   >
     <AnalyticsChart
       :chart-data="data"
       :chart-options="options"
       chart-title=""
+      :height="`${height}px`"
+      legend-position="bottom"
+      :show-annotations="chartOptions.showAnnotations"
       tooltip-title=""
     />
   </QueryDataProvider>
 </template>
-<script setup lang="ts">
+<script setup lang="ts" generic="">
```

Comment:
Minor: remove `generic` attribute for the bar chart renderer and timeseries renderer.